pymc_marketing/mmm/components/saturation.py

```diff
@@ -197,6 +198,39 @@ def function(self, x, lam, beta):
     }
 
 
+class InverseScaledLogisticSaturation(SaturationTransformation):
+    """Wrapper around inverse scaled logistic saturation function.
+
+    For more information, see :func:`pymc_marketing.mmm.transformers.inverse_scaled_logistic_saturation`.
+
+    .. plot::
+        :context: close-figs
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        from pymc_marketing.mmm import InverseScaledLogisticSaturation
+
+        rng = np.random.default_rng(0)
+
+        adstock = InverseScaledLogisticSaturation()
+        prior = adstock.sample_prior(random_seed=rng)
+        curve = adstock.sample_curve(prior)
+        adstock.plot_curve(curve, sample_kwargs={"rng": rng})
+        plt.show()
+
+    """
+
+    lookup_name = "inverse_scaled_logistic"
+
+    def function(self, x, lam, beta):
+        return beta * inverse_scaled_logistic_saturation(x, lam)
+
+    default_priors = {
+        "lam": Prior("Gamma", alpha=3, beta=1),
```

Comment:
Since we scale the x values between zero and 1, does it make sense to have the prior with much of the density between 0 and 1? 
What does this distribution looks like?

Reply:
Maybe something like: 
Gamma(alpha=alpha beta=2 * alpha)

which would put average at 0.5 but choose alpha such that there is most of probability between 0 and 1 (because of the x domain)

Reply:
Maybe something like: 
Gamma(alpha=alpha beta=2 * alpha)

which would put average at 0.5 but choose alpha such that there is most of probability between 0 and 1 (because of the x domain)

This comes with the assumption that the half-way saturation is (likely) below the max spend of the media in training.  

Would be interested in hearing thoughts here!

Reply:
thinking of Gamma(alpha=0.3, beta=0.6), so that:
P(lam>1) = 0.0338 (low enough)
P(lam>0.5) = 0.10811 (not too skewed towards 0)